README.md

```diff
@@ -1,2 +1,56 @@
 # sustainable-society
 Collection of documents to use when needing an advice regarding sustainability 
+
+## Food
+-  Buy local and seasonal food, no need to buy summer food in winter, or if you do have cravings for it buy (or do it yourself) pickled or in naother way preserved version
+
+- Avoid supermarkets for most of the things, buy there only the things that are not available locally
```

Comment:
Avoid supermarkets in general (even if it's not feasible, it's a good guideline).